Modules/IndividualAssessment/classes/class.ilIndividualAssessmentExporter.php

```diff
@@ -54,6 +54,7 @@ public function getValidSchemaVersions(string $a_entity) : array
     {
         return [
             "5.2.0" => [
+                // PHP8-Review: Link with unencrypted protocol
```

Comment:
This is interesting, I don't exactly understand if the protocol is really relevant here. Since this looks like some naming related stuff and does not seem to be an URL someone actually calls, I expect TLS to be irrelevant. On the other hand, switching to `https` might disturb some mechanism. I really don't know. So we might want to discuss this on the PHP 8 meeting.

Reply:
As discussed in the latest PHP 8-Meeting: I will not change this =)